src/formatter.ts

```diff
@@ -772,29 +886,29 @@ export class Formatter {
         }
       }
 
-      shift *= options.alpha;
+      if (options) shift *= options.alpha;
```

Comment:
Silent failure here -- `options.alpha` should always exist.

Reply:
done